packages/office-ui-fabric-react/src/components/TextField/TextField.types.ts

```diff
@@ -192,22 +192,22 @@ export interface ITextFieldProps extends React.AllHTMLAttributes<HTMLInputElemen
   /**
    * Function called after validation completes.
    */
-  onNotifyValidationResult?: (errorMessage: string, value: string | undefined) => void;
+  onNotifyValidationResult?: (errorMessage: string | JSX.Element, value: string | undefined) => void;
 
   /**
    * Function used to determine whether the input value is valid and get an error message if not.
    * Mutually exclusive with the static string `errorMessage` (it will take precedence over this).
    *
-   * When it returns string:
+   * When it returns string | JSX.Element:
    * - If valid, it returns empty string.
-   * - If invalid, it returns the error message string and the text field will
+   * - If invalid, it returns the error message and the text field will
    *   show a red border and show an error message below the text field.
    *
-   * When it returns Promise<string>:
+   * When it returns Promise<string | JSX.Element>:
```

Comment:
There's currently a build error:
```
Warning: lib/components/TextField/TextField.types.d.ts:171:52 - (tsdoc-escape-greater-than) The ">" character should be escaped using a backslash to avoid confusion with an HTML tag
Warning: lib/components/TextField/TextField.types.d.ts:171:31 - (tsdoc-html-tag-missing-greater-than) The HTML tag has invalid syntax: Expecting an attribute or ">" or "/>"
```

Pretty sure this should fix it:
```suggestion
   * When it returns Promise\<string | JSX.Element\>:
```

Reply:
Fixed